android/src/main/java/com/tailscale/ipn/IPNService.java

```diff
@@ -61,6 +62,12 @@ private PendingIntent configIntent() {
 			.setConfigureIntent(configIntent())
 			.allowFamily(OsConstants.AF_INET)
 			.allowFamily(OsConstants.AF_INET6);
+		try {
+			b.addDisallowedApplication("com.tailscale.ipn");
```

Comment:
I'd rather we look this up from ourselves rather than hard code it where it might fall out of sync in a future fork or rename.

Reply:
BuildConfig.APPLICATION_ID

Reply:
@eliasnaur, can you finish this change up and submit it? @danderson is out until the new year.